<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digital Circuit</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="separator"/>
   <addaction name="actionAddNOT"/>
   <addaction name="actionAddAND"/>
   <addaction name="actionAddNAND"/>
   <addaction name="actionAddOR"/>
   <addaction name="actionAddNOR"/>
   <addaction name="actionAddXOR"/>
   <addaction name="actionAddXNOR"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="assets/resources.qrc">
     <normaloff>:/toolbar/start.png</normaloff>:/toolbar/start.png</iconset>
   </property>
   <property name="text">
    <string>StartStop</string>
   </property>
   <property name="toolTip">
    <string>Starts the simulation</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAddAND">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/resources.qrc">
     <normaloff>:/components/and.png</normaloff>:/components/and.png</iconset>
   </property>
   <property name="text">
    <string>AddAND</string>
   </property>
   <property name="toolTip">
    <string>Create an AND gate</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionAddOR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/resources.qrc">
     <normaloff>:/components/or.png</normaloff>
     <selectedoff>:/components/and.png</selectedoff>:/components/or.png</iconset>
   </property>
   <property name="text">
    <string>AddOR</string>
   </property>
   <property name="toolTip">
    <string>Create an OR gate</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionAddNOT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/resources.qrc">
     <normaloff>:/components/not.png</normaloff>:/components/not.png</iconset>
   </property>
   <property name="text">
    <string>AddNOT</string>
   </property>
   <property name="toolTip">
    <string>Create a NOT gate</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionAddNAND">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/resources.qrc">
     <normaloff>:/components/nand.png</normaloff>:/components/nand.png</iconset>
   </property>
   <property name="text">
    <string>AddNAND</string>
   </property>
   <property name="toolTip">
    <string>Create a NAND gate</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionAddNOR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/resources.qrc">
     <normaloff>:/components/nor.png</normaloff>:/components/nor.png</iconset>
   </property>
   <property name="text">
    <string>AddNOR</string>
   </property>
   <property name="toolTip">
    <string>Create a NOR gate</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionAddXOR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/resources.qrc">
     <normaloff>:/components/xor.png</normaloff>:/components/xor.png</iconset>
   </property>
   <property name="text">
    <string>AddXOR</string>
   </property>
   <property name="toolTip">
    <string>Create an XOR gate</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="actionAddXNOR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/resources.qrc">
     <normaloff>:/components/xnor.png</normaloff>:/components/xnor.png</iconset>
   </property>
   <property name="text">
    <string>AddXNOR</string>
   </property>
   <property name="toolTip">
    <string>Create an XNOR gate</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
